context/karma.js

```diff
@@ -118,7 +118,14 @@ function ContextKarma (callParentKarmaMethod) {
         }
         localConsole[method] = function () {
           self.log(method, arguments)
-          return Function.prototype.apply.call(orig, localConsole, arguments)
+          try {
+            return Function.prototype.apply.call(orig, localConsole, arguments)
+          } catch (error) {
+            // Sometimes native console methods are broken, but this shouldn't break tests.
+            // It is especially undesirable in environments like Sauce Labs with limited access to the browser settings.
+            // See https://stackoverflow.com/questions/46913856/value-is-not-a-sequence-safari-exception
+            self.log('warn', ['Native method console.' + method + ' is broken:', error])
```

Comment:
Please use string literal but without attributing the throw to a cause:
```js
`Console method ${method} threw: ${error}`
```

This issue can also arise if users overwrite `console`: it may not be a native method problem.

Reply:
Turns out we can't use ES2015 there. If we do, it doesn't work in Phantom and other old browsers.